probe/tag/pidtree.go

```diff
@@ -24,7 +24,7 @@ var (
 	readFile = ioutil.ReadFile
 )
 
-func newPIDTree(procRoot string) (*pidTree, error) {
+func newRealPIDTree(procRoot string) (*pidTree, error) {
```

Comment:
I would argue for the *Stub as it makes it obvious as to why you are taking a local reference to this function; see Mathias's comment earlier today.